react-components/tests/tests-utilities/fixtures/renderTarget.ts

```diff
@@ -1,7 +1,7 @@
 import { Mock } from 'moq.ts';
 import {
   type CommandsController,
-  type RevealRenderTarget,
+  RevealRenderTarget,
```

Comment:
```suggestion
  type RevealRenderTarget,
```

Will fix linting error